e2e-tests/trade-finance-manager/cypress/integration/journeys/case/activities/activities.spec.js

```diff
@@ -0,0 +1,112 @@
+            activities: {
+              type: 'OTHER',
+              timestamp: 13345665,
+              text: 'Not a comment',
+              author: {
+                firstName: 'tester',
+                lastName: 'smith',
+                _id: 12243343242342,
+              },
+              label: 'Other',
+            },
+          },
+        };
+        cy.updateTFMDeal(dealId, otherActivity);
+        cy.login(businessSupportUser);
+      });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('connect.sid');
+    cy.visit(relative(`/case/${dealId}/activity`));
+  });
+
+  describe('add a comment', () => {
+    it('should go to add a comment page if add comment button clicked', () => {
+      activitiesPage.addACommentButton().click();
+      cy.url().should('eq', relative(`/case/${dealId}/activity/post-comment`));
+    });
+
+    it('entering no comment should take you back to activity page and no comment in timeline', () => {
+      activitiesPage.addACommentButton().click();
+      activityCommentBoxPage.addCommentButton().click();
+      activitiesPage.activitiesTimeline().contains(userFullName).should('not.exist');
+    });
+
+    it('should submit a comment under 1000 characters', () => {
+      const date = format(new Date(), 'dd MMMM yyyy');
+      activitiesPage.addACommentButton().click();
+      activityCommentBoxPage.activityCommentBox().type('test');
+      activityCommentBoxPage.addCommentButton().click();
+      activitiesPage.activitiesTimeline().contains('test');
+      activitiesPage.activitiesTimeline().contains(userFullName);
+      activitiesPage.activitiesTimeline().contains(date);
+    });
+
+    it('pressing cancel should not submit a comment', () => {
+      activitiesPage.addACommentButton().click();
+      activityCommentBoxPage.activityCommentBox().type('should cancel');
+      activityCommentBoxPage.cancelButton().click();
+      activitiesPage.activitiesTimeline().contains('should cancel').should('not.exist');
+    });
+
+    it('pressing comment filter should show comments', () => {
+      activitiesPage.activitiesTimeline().contains('Not a comment');
+      activitiesPage.filterCommentsOnly().click();
+      activitiesPage.activitiesTimeline().contains('test');
+      activitiesPage.activitiesTimeline().contains(userFullName);
+      // ensures that is filtered out
+      activitiesPage.activitiesTimeline().contains('Not a comment').should('not.exist');
+    });
+
+    it('should not be allowed to add comment over 1000 characters', () => {
+      const longComment = 'aaaaaaaaaa'.repeat(101);
+      activitiesPage.addACommentButton().click();
+      activityCommentBoxPage.activityCommentBox().type(longComment);
+      activityCommentBoxPage.addCommentButton().click();
+      activityCommentBoxPage.commentErrorSummary().contains('Comments must be 1000 characters or fewer');
+      activityCommentBoxPage.commentErrorMesssage().contains('Comments must be 1000 characters or fewer');
```

Comment:
duplicate here, not a blocker

Reply:
I was testing both the error summary and error message, since warning twice? should I just do 1?

Reply:
Sorry didn't notice the different selectors, perfect! 👍